packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -915,7 +919,23 @@ export class ContainerRuntime extends EventEmitter
     public async resolveHandle(request: IRequest): Promise<IResponse> {
         const requestParser = new RequestParser(request);
 
-        if (requestParser.pathParts.length > 0) {
+        if (requestParser.pathParts.length > 0 && requestParser.pathParts[0] === "blobs") {
```

Comment:
Please consider having "blobs" variable somewhere and using it here and in blob manager (for example it can be readonly property of blobManager), such that it's more obvious that both places are connected.  
Also I'd suggest to use _blobs, less chances of collision with existing IDs (as in many _something naming is reserved for "system")